local-testing-docs/README.md

```diff
@@ -267,34 +267,525 @@ Where all tables created by `DataFrame.save_as_table` are saved as temporary tab
 As explained in [Installation](#installation), updates will be pushed periodically to the branch `dev/local-testing`, so if you want to use those recent changes you will need to re-run the `pip install ...` or `conda env update ...` commands to re-install the Snowpark package from the branch.
 
 ## Supported APIs
+<details>
+  <summary>Column</summary>
+
+- Column.alias
+- Column.as_
+<!--
+Column.asc
+Column.asc_nulls_first
+Column.asc_nulls_last
+Column.astype
+Column.between
+Column.bitand
+Column.bitor
+Column.bitwiseAnd
+Column.bitwiseOR
+Column.bitwiseXOR
+Column.bitxor
+Column.cast
+Column.collate
+Column.desc
+Column.desc_nulls_first
+Column.desc_nulls_last
+Column.endswith
+Column.eqNullSafe
+Column.equal_nan
+Column.equal_null
+Column.getItem
+Column.getName
+Column.get_name
+Column.in_
+Column.isNotNull
+Column.isNull
+Column.is_not_null
+Column.is_null
+Column.isin
+Column.like
+Column.name
+Column.over
+Column.regexp
+Column.rlike
+Column.startswith
+Column.substr
+Column.substring
+Column.try_cast
+Column.within_group
+CaseExpr.when
+CaseExpr.otherwise
+-->
+</details>
+
+<details>
+  <summary>DataFrame</summary>
+
+- DataFrame.agg <!--
```

Comment:
Feel free to point out items that are supported already. We should update this file as feature gets supported.